lib/phoenix_live_view/html_formatter.ex

```diff
@@ -418,22 +436,28 @@ defmodule Phoenix.LiveView.HTMLFormatter do
   end
 
   defp to_tree([{:tag_close, name, _meta} | tokens], buffer, [{name, attrs, upper_buffer} | stack]) do
-    mode =
+    {mode, attrs} =
       cond do
         name in ["pre", "textarea"] ->
-          :preserve
+          {:preserve, attrs}
 
         name in @inline_elements and head_is_text_ending_with_whitespace?(upper_buffer) ->
-          :preserve
+          {:preserve, attrs}
 
         contains_special_attrs?(attrs) ->
-          :preserve
+          attrs =
+            Enum.reject(attrs, fn
+              {"phx-no-break", _v} -> true
```

Comment:
```suggestion
              {"phx-no-break", _v} -> true
```

we Cannot remove the attribute in the formatter, because next time the attribute with no longer be there. We have to remove it in the engine, wire Henry